### DISCUSSION TITLE
Display blade file name instead of view cache file name.

### DISCUSSION THREAD

**Discussion by muath-ye** [spatie/laravel-ignition#95] [Q&A]
I can not recognized the blade file which has the error, because it shows me the cache file name

![image](https://user-images.githubusercontent.com/34031333/172129599-fcc96293-3ff4-465c-bb68-e4dd01718cde.png)

**Comment by jcmcclorey** [comment]
We can't be the only 2 that find this annoying. Is there any kind of solution or explanation as to why it works this way now instead of how it used to work in facade/ignition?

**Comment by jcmcclorey** [comment]
I have just opened a pending Pull Request #122 to fix this issue (#104)
